ElastosBountyProgram/back-end/src/service/CommentService.ts

```diff
@@ -35,15 +35,28 @@ export default class extends Base {
 
             const mentions = comment.match(/@\w+/g)
             if (mentions) {
-                this.sendMentionEmails(type, param, createdBy, mentions, returnUrl)
+                this.sendMentionEmails(type, param, createdBy, mentions, returnUrl, commentable.name)
```

Comment:
First of all, this will send those emails twice, but regardless - I think passing .name in like you do here is enough, the check below for ```if type === 'Task'``` shouldnt be needed (if name is not on the commentable, just dont display it - sounds simple enough).